File: Scala/abc197_a/Main.scala
@main def Main(): Unit =
  val lines = scala.io.StdIn.readLine()
  val result = solve(lines)
  println(result)

def solve(input: String): String =
  val x = input.substring(0, 1)
  val xs = input.substring(1, input.length)
  xs + x

File: Golang/abc197_a/main.go
package main

import (
	"bufio"
	"fmt"
	"os"
)

func read_line() string {
	scanner := bufio.NewScanner(os.Stdin)
	scanner.Scan()
	text := scanner.Text()
	return text
}

func main() {
	text := read_line()

	result := solve(text)

	fmt.Println(result)
}

func solve(s string) string {

	x := s[:1]
	xs := s[1:]

	return xs + x
}
